_config.yml

```diff
@@ -21,8 +21,11 @@ description: "Towards an Infrastructure for the Distributed Exploration and Anno
 
 # This URL is the main adresse. Don't include a slash at the end.
 #
-url: "//inception-project.github.io/"
+url: "https://inception-project.github.io"
```

Comment:
Hm - this might not be relevant anymore but at some point, we intentionally removed the protocol such that intra-site links use the protocol that the user put into the browser... If there is no need to add the protocol name, I'd continue going without it.

Reply:
Github redirects to https automagically